config/initializers/rack_rewrite.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+# URL for Nuts Server deployed on Heroku, redirects download/update requests to latest release
+# on GitHub Releases with platform parameters
+RELEASE_SERVER_URL = "http://classroom-desktop-releases.herokuapp.com"
```

Comment:
`s/desktop/assistant`

Reply:
@johndbritton Great catch, could you rename the Heroku app from `classroom-desktop-releases` -> `classroom-assistant-releases`? I don't have admin access on the app